microsoft-edge/progressive-web-apps-chromium/how-to/webappmanifests.md

```diff
@@ -0,0 +1,109 @@
+---
+title: Use a Web App Manifest to integrate a Progressive Web App into the Operating System
+description: Learn how to use the Web App Manifest to integrate your Progressive Web App into your operating system.
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.date: 01/07/2021
+ms.topic: conceptual
+ms.prod: microsoft-edge
+ms.technology: pwa
+keywords: progressive web apps, PWA, Edge, JavaScript, Windows, UWP, Microsoft Store
+---
+# Use a Web App Manifest to integrate a Progressive Web App into the Operating System
+
+<!-- todo: above the Shortcuts section, add basics/code, like in the workshop or MDN content overview of properties (want the barebones that are needed to get started).  The workshop walks through effects (slide 55 "A little color").  The workshop might be helpful to watch when writing, to explain the bits & pieces.  Walks through web app manifests & service workers. -->
```

Comment:
@captainbrosset - Requested by SME in June 22 meeting: 
Above the Shortcuts section, add basics/code, like in the workshop or MDN content overview of properties.  Add the barebones code blocks that are needed to get started.  The workshop walks through effects (slide 55 "A little color"). The workshop might be helpful to watch when writing, to explain the bits & pieces.  It walks through web app manifests & service workers.
https://noti.st/aarongustafson/co3b5z/getting-started-with-progressive-web-apps-workshop
(Known issue: Final Demo links are 404)

Reply:
Thanks for the note. I've added some content for this now.